sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,63 @@
+-- Kologarn
+UPDATE `creature_template` SET `unit_flags`=0, `unit_flags2`=34816 WHERE `entry` IN (32930,33909);
+UPDATE `creature_template` SET `unit_flags`=0, `unit_flags2`=133120, `RegenHealth`=0 WHERE `entry` IN (32934,32933,33911,33910);
+UPDATE `creature_template` SET `ScriptName`='npc_focused_eyebeam' WHERE `entry` IN (33632,33802);
+UPDATE `creature_template_movement` SET `Ground`=1, `Flight`=0, `Rooted`=1 WHERE `CreatureId` IN (32930,33909);
+DELETE FROM `creature` WHERE `id`=32930;
+DELETE FROM `creature_addon` WHERE `guid`=136762;
+
+DELETE FROM `creature_template_movement` WHERE `CreatureId`=33661;
+INSERT INTO `creature_template_movement` (`CreatureId`, `Ground`, `Swim`, `Flight`, `Rooted`) VALUES
+(33661,0,0,0,1);
+
+DELETE FROM `creature_template_addon` WHERE `entry` IN(32930,33909,32934,32933);
+INSERT INTO `creature_template_addon` (`entry`,`path_id`,`mount`,`bytes1`,`bytes2`,`emote`,`auras`) VALUES
+(32934,0,0,0,1,0,64651),
+(32933,0,0,0,1,0,64651),
+(32930,0,0,0,1,0,64651),
+(33909,0,0,0,1,0,64651);
```

Comment:
"64651"

Reply:
Oh. Right. `auras` are `text` type, not int.